dashboards-reports/server/routes/utils/notification/notification_content_template/email_content_template.html

```diff
@@ -20,7 +20,7 @@
 
       body {
         background-color: #f6f6f6;
-        font-family: "Inter UI", -apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+        font-family: "Open Sans", "Segoe UI", Tahoma, sans-serif;
```

Comment:
just curious why did the font change? I see the previous `font-family` combination in dashboards a lot

Reply:
Couple of reasons. The new brand uses Open Sans, and becuase this is email, this fits the brand. The second reason is that this new font stack is available in more systems out of the box. We're not really using emojis and symbols in the email, so those are redundant. Inter UI is not a common system font.

Reply:
got it, thanks